.github/sync-repo-settings.yaml

```diff
@@ -66,6 +66,22 @@ branchProtectionRules:
       - Kokoro - Against Pub/Sub Lite samples
       - cla/google
       - OwlBot Post Processor
+  - pattern: lts-1.116.x
```

Comment:
6 months ago, it was "1.114.x" branch. You may want to stick the pattern. That is, "1.116.x".

Reply:
Will do, I am running into some errors but will try again with that naming pattern